docs/docs/glossary.mdx

```diff
@@ -21,7 +29,15 @@ description: Glossary for all Rasa-related terms
 
 ## CMS
 
-  A Content Management System (CMS) can be used to store bot responses externally instead of directly including it as part of the domain.  This provides more flexibility in changing them as they are not tightly-coupled with the training data.
+  Can be used to store bot responses externally instead of including them directly in the domain. Content Management Systems decouple responses from training data, which can provide greater flexibility when making updates
+  
+## Conversation-Driven Development (CDD)
+
+  The process of using real user messages and conversations to influence the design of an assistant and train the model, combined with engineering best practices. There are 6 steps that make up CDD: Share, Review, Annotate, Fix, Track, and Test.
```

Comment:
hmm, to me, "listening to your users" is a bit too vague for a glossary. I think it's ok in the longer article because there's a lot of additional material, but in a 1-2 sentence definition, it feels fuzzy. Did you have thoughts on the wording?